src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -277,16 +266,20 @@ namespace AirflowNetworkSolver {
         }
 
         for (i = 1; i <= NetworkNumOfNodes; ++i) {
-            TZ(i) = AirflowNetworkNodeSimu(i).TZ;
-            WZ(i) = AirflowNetworkNodeSimu(i).WZ;
+            // TZ(i) = AirflowNetworkNodeSimu(i).TZ;
+            // WZ(i) = AirflowNetworkNodeSimu(i).WZ;
             PZ(i) = AirflowNetworkNodeSimu(i).PZ;
+            properties[i].temperature = AirflowNetworkNodeSimu(i).TZ;
+            properties[i].humidityRatio = AirflowNetworkNodeSimu(i).WZ;
+            // properties[i].pressure = AirflowNetworkNodeSimu(i).PZ;
```

Comment:
DO you plan to remove PZ(i) = AirflowNetworkNodeSimu(i).PZ; and use properties[i].pressure = AirflowNetworkNodeSimu(i).PZ;?

Reply:
Yes, at some point. It's a change that goes with other changes that end up causing diffs, so that'll come in a later PR.